js/map.js

```diff
@@ -1,230 +1,200 @@
-var map;
-var base_layer;
-var comm_layer;
-var comm_sublayer;
-var parcel_layer;
-var parcel_sublayer;
-var info;
-var mapColors = [
-  '#a50f15',
-  '#de2d26',
-  '#fb6a4a',
+const mapColors = [
+  '#000000',
+  '#fee5d9',
   '#fcae91',
-  '#fee5d9'
-];
-
-var commAreaTotal = [ 292506638.00, 159211446.00, 103827021.00, 40812087.00, 1238123.00 ];
-var commAreaNonviolent = [ 54310585.00, 28117076.00, 15046737.00, 6392911.00, 719085.00 ];
-var commAreaViolent = [ 105575382.00, 56782734.00, 26092813.00, 9832279.00, 116435.00 ];
-var commAreaDrug = [ 132620671.00, 91139090.00, 41372510.00, 11930226.00, 40051.00 ];
-
-var parcelTotal = [ 16512329.00, 3829048.00, 1314546.00, 426732.00, 14.00 ];
-var parcelNonviolent = [ 6988883.00, 1899220.00, 597267.00, 168647.00, 1.00 ];
-var parcelViolent = [ 5187690.00, 1889079.00, 663269.00, 176069.00, 1.00 ];
-var parcelDrug = [ 2003918.00, 727010.00, 304011.00, 92846.00, 1.00 ];
-
-function init(type){
-  // initiate leaflet map
-  if (map === undefined){
-    map = new L.Map('cartodb-map', {
-      center: [41.8650, -87.6656],
-      zoom: 12,
-      scrollWheelZoom: false
-    });
-  }
-
-  var jenks_values_commarea = commAreaTotal;
-  var jenks_values_parcel = parcelTotal;
-  var data_column = 'total_cost';
-  var type_name = 'Total';
+  '#fb6a4a',
+  '#de2d26',
+  '#a50f15',
+]
+
+const mapOpacities = [
+  0.1,
+  0.9,
+  0.9,
+  0.9,
+  0.9,
+]
+
+const geography_buckets = {
+  'blocks':           { 'total': [0, 426732, 1314546, 3829048, 16512329],
+                        'nonviolent': [0, 168647, 597267, 1899220, 6988883],
+                        'violent': [0, 176069, 663269, 1889079, 5187690],
```

Comment:
note that violent and nonviolent layers are not used, but i kept the buckets in here for posterity